components/sup/src/main.rs

```diff
@@ -170,9 +170,10 @@ fn sub_run(m: &ArgMatches, launcher: LauncherCli) -> Result<()> {
                         .as_ref()
                         .unwrap_or(&*protocol::DEFAULT_BLDR_URL),
                     &source,
-                    msg.bldr_channel
+                    &msg.bldr_channel
                         .as_ref()
-                        .unwrap_or(&*protocol::DEFAULT_BLDR_CHANNEL),
+                        .map(|s| ChannelIdent::from(s))
+                        .unwrap_or_default(),
```

Comment:
This is another instance where I don't think we want to let an environment variable value sneak in.

Reply:
Will add to the list for follow up